ccd-definition/AuthorisationCaseField/AuthorisationCaseField-JO-nonprod.json

```diff
@@ -143,7 +129,6 @@
   "CaseTypeID": "CIVIL",
   "CaseFieldID": "joJudgmentPaidInFull",
```

Comment:
Is this field is related to "Rcord or Eduit judgement". I think claimant can also mark it as paid in full thats why I would suggest not modifying permissions for fields "judgement paid in full"

Reply:
Yes, solicitors, etc. should be able to mark paid in full, I'll change it back.